features/aanschrijfwijze.feature

```diff
@@ -147,72 +170,73 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
       | fields              | naam.aanschrijfwijze.naam       |
-      Dan bevat de persoon met burgerservicenummer '999992934' de volgende 'naam' gegevens
+      Dan heeft de response een persoon met de volgende 'naam' gegevens
       | naam                 | waarde                    |
       | aanschrijfwijze.naam | <naam in aanschrijfwijze> |
 
       Voorbeelden:
-      | geslacht | voornamen | voorvoegsel | geslachtsnaam | titel/predicaat | naam in aanschrijfwijze    |
-      | M        | Franklin  | van den     | Aedel         | JH              | Jonkheer F. van den Aedel  |
-      | V        | Francisca |             | Groenen       | JV              | Jonkvrouw F. Groenen       |
-      | M        | Franklin  |             | Groenen       | JV              | Jonkheer F. Groenen        |
-      | V        | Francisca | van den     | Aedel         | JH              | Jonkvrouw F. van den Aedel |
-      | M        | Franklin  | van den     | Aedel         | R               | F. ridder van den Aedel    |
-      | V        | Francisca | van den     | Aedel         | R               | F. van den Aedel           |
-      | M        | Franklin  | van den     | Aedel         | B               | F. baron van den Aedel     |
-      | V        | Francisca |             | Groenen       | BI              | F. barones Groenen         |
-      | M        | Franklin  | van den     | Aedel         | H               | F. hertog van den Aedel    |
-      | V        | Francisca |             | Groenen       | HI              | F. hertogin Groenen        |
-      | M        | Franklin  | van den     | Aedel         | G               | F. graaf van den Aedel     |
-      | V        | Francisca | van den     | Aedel         | GI              | F. gravin van den Aedel    |
-      | M        | Franklin  | van den     | Aedel         | M               | F. markies van den Aedel   |
-      | V        | Francisca | van den     | Aedel         | M               | F. markiezin van den Aedel |
-      | M        | Franklin  | van den     | Aedel         | P               | F. prins van den Aedel     |
-      | V        | Francisca | van den     | Aedel         | PS              | F. prinses van den Aedel   |
-      | M        | Franklin  | van den     | Aedel         | PS              | F. prins van den Aedel     |
-      | V        | Francisca | van den     | Aedel         | P               | F. prinses van den Aedel   |
-      | O        | Franklin  | van den     | Aedel         | M               | F. van den Aedel           |
+      | geslacht | voornamen | voorvoegsel | geslachtsnaam | titel/predicaat | naam in aanschrijfwijze    | opmerkingen                                                                                    |
+      | M        | Franklin  | Van Den     | Aedel         | JH              | Jonkheer F. van den Aedel  | het voorvoegsel wordt met kleine letters opgenomen                                             |
```

Comment:
de rule op regel 48 "het voorvoegsel wordt met kleine letters opgenomen" is niet consistent met hoe dit in aanhef en gebruik_in_lopende_tekst is gespecificeerd. We hebben hierover een beslissing genomen in https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/922#issuecomment-1054283258

Wat dan wel blijft is wat op regel 47 staat; "de eerste naamcomponent in de aanschrijfwijze.naam begint met een hoofdletter". Dat kan op voorletters van toepassing zijn wanneer de persoon geen voornamen heeft.